### ISSUE TITLE
Want to know optimizer and scheduler 

### ISSUE THREAD

**Issue by mandy0409** [lisiyao21/AnimeInterp#22]
Since I am really impressed about your work, I am implementing the training code of the paper .

However, I couldn't find any information about optimizer in the paper. 

Could you tell me what optimizer and scheduler you used when you train the RFR model and whole model?

**Comment by lisiyao21** [comment]
Thanks for your interests!

We use AdamW to train the RFR with following settings:
1. FlyingChairs: bsz=8, lr=0.00025, num_steps=120000, image_size 368 496 --wdecay 0.0001
2. FlyingThings: bsz=5, lr=0.0001, num_steps=120000, image_size 400 720 --wdecay 0.0001
3. Sintel: bsz=5, lr=0.0001, num_steps=120000, image_size 368 768 --wdecay 0.0001

Check the pretrained RFR weight at #21  if you need.

For training the whole pipeline, we use Adam without weight decay. The concrete parameters are as written in our paper.

**Comment by mandy0409** [comment]
Thank you for your response!

**Comment by mandy0409** [comment]
Oh, I have a question. Didn't you use any schedulers when you are training?
